src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesSynchronisationService.kt

```diff
@@ -251,9 +249,7 @@ Also add new mappings for the new booking id for the copied case notes, which po
       val newMappings =
         freshlyMergedNomisCaseNotes.map { newNomisCaseNote ->
           CaseNoteMappingDto(
-            dpsCaseNoteId = existingMappings.find {
-              it.nomisCaseNoteId == newToOldMap[newNomisCaseNote.caseNoteId]?.caseNoteId
-            }
+            dpsCaseNoteId = existingMappings[newToOldMap[newNomisCaseNote.caseNoteId]?.caseNoteId]
```

Comment:
can go directly into our `existingMappings` map now to grab the dps case note id rather than having to do a `find`